docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       PGDATABASE: procrastinate
       PGHOST: postgres
       PGUSER: postgres
+      POSTGRES_PASSWORD: password
```

Comment:
This should actually be set at postgres service level.

Reply:
you mean alongside the variable `POSTGRES_DB` ? It's not a environment variable ?

Reply:
You need `PGPASSWORD` in the procrastinate env vars (in the same way we set `PGPASSWORD` for `psql`), and `POSTGRES_PASSWORD` in the postgres env vars.

Reply:
it should be done thank you :)